src/MongoDB.Driver.Core/Core/Servers/Server.cs

```diff
@@ -31,6 +31,7 @@
 using MongoDB.Driver.Core.Connections;
 using MongoDB.Driver.Core.Events;
 using MongoDB.Driver.Core.Misc;
+using MongoDB.Driver.Core.Servers.Strategies;
```

Comment:
this class almost have **no changes** other than the monitor related logic was moved into `DefaultServerStrategy` class and small simplification of how we work with error codes